### ISSUE TITLE
Node 10 not working

### ISSUE THREAD

**Issue by xzyfer** [sass/node-sass#2345]
## tl;dr
```
npm install node-sass@beta
```

*****

We know. Trust us, we know. We're working on it.

## Blockers

We're cannot proceed until appveyor/ci#2272 is fixed. (**edit**: now fixed)

## Prerequisites 

First things first, you'll **need to have [node-sass@4.9.0][1]**.

[1]: https://github.com/sass/node-sass/releases/tag/v4.9.0

## node-sass@4.9.0 isn't published yet!

See **blockers** above.

It's night time in Australia, us koalas are sleeping.

Try again tomorrow. May be complain Twitter to feel better. We're [@NodeSass][2], like and subscribe 💖 

[2]: https://twitter.com/nodesass

## What version am I using?

```
npm ls node-sass
```

### npm

```
npm install node-sass@latest
```

If node-sass is a dependency of another library
- delete your `node_modules`
- run `npm install`

### yarn

You'll need to do an update to override your lock file.

```
yarn upgrade --latest node-sass
```

## When will Node 10 support on my environment?

Here is are the current support environments. Click the `subscribe` button next to this issue to get updates.

| OS | Architecture | Node |
| --- | --- | --- |
| Windows | x86, x64 | ✅  |
| OSX | x64 | ✅  |
| Linux* | x86 | ❓  |
| Linux* | x64 | ✅ |
| Alpine Linux | x64 | ✅  |
| FreeBSD 10+ | amd64 | ❌  |
| FreeBSD 10+ | i386 | ❌  |

*Linux support refers to Ubuntu, Debian, and CentOS 6+
❓There is no x86 build for Node 10 currently available

## Do I need to comment or open another issue?

No.

**Comment by xzyfer** [comment]
## Update

### LibSass

We've run into a few hiccups.

Some regressions in the LibSass@3.5.3 update were caught early.
We've released LibSass@3.5.4 to address these. 
LibSass@3.5.4 will be in node-sass@v4.9.0.

#2348

### Node 10

Node 10 support has landed in Travis and AppVeyor.

However Node 10 has changed their minimum required gcc version. 
We're working to update our build pipelines.

#2349

**Comment by xzyfer** [comment]
## Update

We believe all issues have been resolved.

We've created the v4.9.0 tag, and have begun building the binaries.

We plan to release v4.9.0 under the `beta` tag within 24hrs.

**Comment by xzyfer** [comment]
No news is good news I suppose?

**Comment by xzyfer** [comment]
node-sass@v4.9.0 is now the stable release.

**Comment by huenisys** [comment]
Cannot download https://github.com/sass/node-sass/releases/download/v4.10.0/win32-x64-64_binding.node

**Comment by schallm** [comment]
Receiving following when try and using this package on Windows:

Cannot download "https://github.com/sass/node-sass/releases/download/v4.11.0/win32-x64-72_binding.node"
